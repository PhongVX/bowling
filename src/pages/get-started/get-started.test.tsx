import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import { GetStartedPage } from ".";

describe("GetStartedPage Snapshot", () => {
  it("renders correctly and matches snapshot", () => {
    const { container } = render(
      <BrowserRouter>
        <GetStartedPage />
      </BrowserRouter>
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
