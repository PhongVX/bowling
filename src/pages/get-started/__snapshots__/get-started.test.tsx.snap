// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GetStartedPage Snapshot renders correctly and matches snapshot 1`] = `
<div
  class="get-started-wrapper"
  style="background-repeat: no-repeat; background-position: center; background-size: cover;"
>
  <div
    class="get-started-center-content"
  >
    <h1
      class="heading get-started-title"
    >
      Bowling
    </h1>
    <button
      class="ant-btn css-dev-only-do-not-override-1yacf91 ant-btn-primary ant-btn-color-primary ant-btn-variant-solid ant-btn-lg"
      test-id="start-game-button"
      type="button"
    >
      <span>
        Start game
      </span>
    </button>
  </div>
</div>
`;
